### PR TITLE
Add last read indicator

### PR DESCRIPTION
**Pull request by gdpelican**
Adds last read comment indicator.

Now with super special bonus lintel indicator as well!

Screenshot:
screenshot from 2015-03-31 11 21 16

### CODE REVIEW

**Comment by rdbartlett** [comment]
pretty cool! just checked it out locally. 

thoughts: 

* should not display if there are no unread items.
* I don't understand the rationale for adding the caret to the lintel? Lintel does one thing well, unread line does one thing well, smushing the two together makes me feel confused about both.

**Comment by rdbartlett** [comment]
Also I think this somehow broke it because I can see the lintel when I'm on the dashboard.

**Comment by gdpelican** [comment]
Hrm, never saw that dashboard lintel; weird. Anyway I took the caret out; I find it really useful (how much of this thing have I read so far?) but understand that it's a little difficult to explain what it is easily. Maybe we'll iterate it in at some point.

**Comment by robguthrie** [comment]
I'm happy to consider iterating it at some point. Thanks James.

File: lineman/app/components/thread_page/activity_card/activity_card.scss [hunk 1]
```diff
@@ -6,4 +6,14 @@
   @include metaText;
   line-height: 30px;
   span{text-decoration: underline;}
-}
\ No newline at end of file
+}
+
+.activity-card-last-read {
```
Comment (robguthrie) [inline lineman/app/components/thread_page/activity_card/activity_card.scss]: Hey James.. I'm trying to move the SCSS to pretty strict BEM syntax. Here Block is activity-card and Element is last-read.. so should be separated with double underscore. Just a note for next time.

Comment (robguthrie) [inline lineman/app/components/thread_page/activity_card/activity_card.scss]: I promise it gets nice to read after a while

**Pull request merged**